Add RTCAudioSourceStats (#26667)
mdn/content: files/en-us/web/api/rtcaudiosourcestats/kind/index.md

---
title: "RTCAudioSourceStats: kind property"
short-title: kind
slug: Web/API/RTCAudioSourceStats/kind
page-type: web-api-instance-property
browser-compat: api.RTCAudioSourceStats.kind
---

{{APIRef("WebRTC")}}

The {{domxref("RTCAudioSourceStats")}} dictionary's property **`kind`** is a string value that is used to differentiate `audio` and `video` media sources.

> Along with the {{domxref("RTCAudioSourceStats.type", "type")}}, this uniquely identifies the type of statistics as a {{domxref("RTCAudioSourceStats")}} object when iterating the {{domxref("RTCStatsReport")}} returned by {{domxref("RTCRtpSender.getStats()")}} or {{domxref("RTCPeerConnection.getStats()")}}.

Reviewer: ```suggestion
Along with the {{domxref("RTCAudioSourceStats.type", "type")}}, this identifies the object as an {{domxref("RTCAudioSourceStats")}} object when iterating the {{domxref("RTCStatsReport")}} returned by {{domxref("RTCRtpSender.getStats()")}} or {{domxref("RTCPeerConnection.getStats()")}}.
```

(not sure what "uniquely" means here)

Author: That's because "uniquely" wasn't necessary. Thanks for the catch.
